src/packages/cli/src/Init.ts

```diff
@@ -138,15 +127,23 @@ export class Init implements Command {
       fs.mkdirSync(outputDir)
     }
 
-    if (!fs.existsSync(prismaFolder)) {
-      fs.mkdirSync(prismaFolder)
-    }
-
     fs.writeFileSync(
-      path.join(prismaFolder, 'schema.prisma'),
+      path.join(outputDir, 'schema.prisma'),
       defaultSchema(provider),
     )
-    fs.writeFileSync(path.join(prismaFolder, '.env'), defaultEnv(url))
+    const envPath = path.join(outputDir, '.env')
+    if (!fs.existsSync(envPath)) {
+      fs.writeFileSync(envPath, defaultEnv(url))
+    }  else {
+      const envFile = fs.readFileSync(envPath, { encoding: 'utf8'})
+      const config = dotenv.parse(envFile) // will return an object
+      if(Object.keys(config).includes("DATABASE_URL")){
+        console.warn(`${chalk.yellow('warn')} DATABASE_URL already exists in ${chalk.bold(envPath)}`)
```

Comment:
Let's be more actionable here and tell users what we would have printed and what they can do

Reply:
What do you mean by this? This is the information that is currently printed https://github.com/prisma/prisma/blob/williamluke4/issue3721/src/packages/cli/src/__tests__/__snapshots__/init.test.ts.snap#L49-L66

Reply:
Something like:
```
To resolve this, we recommend moving the `DATABASE_URL` env var to the root .env
```

Reply:
They don't really need to resolve anything as this is just a warning them that they already have an ENV VAR named DATABASE_URL in their root `.env`. Therefore we will just not add our default DATABASE_URL to it.